src/fileseq/filesequence.py

```diff
@@ -434,3 +434,14 @@ def getPaddingChars(num):
             return "#" * (num / 4)
         else:
             return "@" * num
+
+    @staticmethod
+    def getPaddingNum(chars):
+        """
+        Given a supported group of padding characters, return the amount of padding.
+        :param chars: a supported group of padding characters (str)
+        :return: int
+        """
+        mapping = {'@': 1, '#': 4}
+
+        return sum([mapping[char] for char in chars])
```

Comment:
This is going to bubble up a random `KeyError` when there are unrecognized characters in `chars`. The `KeyError` has no context in this call and isn't documented. A user would have to discover the exception by chance. Either this function should document that it can raise a specific exception, such as a `ValueError` if a bad `chars` value is passed, or you can make this never fail and do something like this

```python
sum((chars.count(c) * size) for c, size in mapping.iteritems())
```